frontend/src/pages/Contact.js

```diff
@@ -1,11 +1,39 @@
 import React from "react"
 import ContactForm from "../components/ContactForm"
+import { Typography, Stack, Container } from "@mui/material"
 
 const Contact = () => {
   return (
-    <div>
-      <h1> Contact page</h1>
-      <ContactForm />
+    <div
+      style={{
+        backgroundColor: "#f4f4f4",
+        height: "150%",
```

Comment:
Was it intentional that the height is greater than 100%? It means that you can scroll down on the page, but there is no content to scroll down on

Reply:
O, it's not suppose to be. It should definitely be only 100%. I think on a few of the other pages, it was suppose to be 150%. 
Good catch

Reply:
I've made changes where all the heights were set to `150%` and changed them to `100%`